src/Chapter1.hs

```diff
@@ -568,8 +572,8 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
-
+isVowel :: Char -> Bool
+isVowel = (`elem` "aeiou")
```

Comment:
Nice one! 👍🏼 

One note in here, that sometimes, `elem` could be slower than the explicit pattern matching. I remember there were some benchmarks on one particular case, that showed how moving to pattern matching on each case separately drastically decrease time 🐎

Reply:
that makes sense, thanks for the information